packages/trader/src/Modules/Trading/Components/Form/TradeParams/Turbos/barriers-list.jsx

```diff
@@ -0,0 +1,64 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const BarriersList = ({
+    active_item_classname,
+    base_classname,
+    selected_item,
+    className,
+    hover_item_classname,
+    list,
+    onClickCb,
+    onMouseEnterCb,
+    onMouseLeaveCb,
+}) => {
+    const onMouseEnter = e => {
+        if (e.target.id && selected_item !== e.target.id) {
+            onMouseEnterCb(e.target.id);
+        }
+    };
+
+    const onMouseLeave = () => {
+        onMouseLeaveCb();
+    };
+
+    const onClick = e => {
+        e.stopPropagation();
```

Comment:
@kate-deriv what do we achieve by using e.stopPropagation(); here?

Reply:
I read that it's a good practice to stop event propagation in order not to trigger event listeners above. But no problem, I'll remove that string